internal/alb/ls/listener.go

```diff
@@ -150,34 +151,49 @@ func NewCurrentListener(o *NewCurrentListenerOptions) (*Listener, error) {
 	}, nil
 }
 
+func (l *Listener) resolveDefaultBackend(rOpts *ReconcileOptions) (*elbv2.Action, error) {
+	if l.defaultBackend.ServicePort.String() == action.UseActionAnnotation {
+		if l.defaultBackend.ServiceName == Default404 {
```

Comment:
i think it might be better to move this logic into where we build annotations for ingress. (e.g. if there user doesn't specify default404 action, we will insert the default one into parsed annotation, otherwise, you use user specified one), so that customer can override our 404 action 😄

Reply:
@bigkraig @M00nF1sh I have exactly this issue. I want to override the default 404 action with a different one and/or produce a different error code? Is there a way to do it? I didn't find any related annotation or any other way.